specs/system_info.table

```diff
@@ -0,0 +1,8 @@
+table_name("system_info")
+description("System information for identification.")
+schema([
+    Column("hostname", TEXT, "Hostname"),
+    Column("hostuuid", TEXT, "UUID used by osquery to uniquely identify this host"),
+    Column("cpu_id", TEXT, "CPU ID"),
```

Comment:
This needs more clarification. Technically this data is the CPU serial, the ID is an overloaded term that most commonly refers to a textual representation of the make/model/revision. It's fine calling it cpu_id but please add some helpful description mentioning that it's the serial. IMO I'd call it cpu_serial though. :)